core/src/main/scala/org/bitcoins/core/gcs/GolombFilter.scala

```diff
@@ -170,7 +172,7 @@ object BlockFilter {
     val (size, filterBytes) = bytes.splitAt(1)
     val n = CompactSizeUInt.fromBytes(size)
 
-    GolombFilter(blockHash.bytes.take(16),
+    GolombFilter(SipHashKey(blockHash.bytes.take(16)),
```

Comment:
There is probably a more acceptable way to do this but this is how I got it to work for now

Reply:
That's actually perfect, although I'd recommend you use an intermediate val like you did above on line 152 to make this more readable